packages/core/src/main/conversation/PayloadBundle.ts

```diff
@@ -70,7 +69,7 @@ interface PayloadBundle {
   id: string;
   state: PayloadBundleState;
   timestamp: number;
-  type: GenericMessageType | CONVERSATION_EVENT | USER_EVENT;
+  type: string;
```

Comment:
Why not `Account.INCOMING`?

Reply:
Because `Account.INCOMING` cannot be used as a type (TypeScript error).

Reply:
Then we will need to refactor `INCOMING` as an enum.